assets/index.ts

```diff
@@ -1,6 +1,23 @@
 import {isTouchDevice} from 'utils';
-import 'primereact/resources/primereact.min.css';
+import {newshubApi, INewshubApi} from './api';
 
 if (isTouchDevice()) {
     document.documentElement.classList.add('no-touch');
 }
+
+let started = false;
+
+export function startApp(): Promise<INewshubApi> {
+    if (started === true) {
+        return Promise.resolve(newshubApi);
+    }
+
+    started = true;
+    return Promise.resolve(newshubApi);
+}
+
+setTimeout(() => {
```

Comment:
I'm wondering what's the usecase for this startApp to be called? just to set the `started` flag?